allennlp/models/model.py

```diff
@@ -100,7 +100,7 @@ def forward(self, input1, input2, targets=None):
         """
         raise NotImplementedError
 
-    def forward_on_instance(self, instance: Instance) -> Dict[str, numpy.ndarray]:
+    def forward_on_instance(self, instance: Instance, cuda_device: int) -> Dict[str, numpy.ndarray]:
```

Comment:
Is `predict` the only command that uses this method?  Does `evaluate` use it?  Or are there any changes necessary in the server code to handle this?

Reply:
One day @matt-gardner, I'll learn the fact that Python is an interpreted language.